client/my-sites/customer-home/cards/features/quick-start/style.scss

```
@@ -0,0 +1,3 @@
+.support-card p {
```

I think this refers to the wrong card.
```suggestion
.quick-start p {
```